tests/queries/0_stateless/03200_memory_engine_alter_dynamic.sql

```diff
@@ -0,0 +1,7 @@
+set allow_experimental_dynamic_type=1;
+create table test (d Dynamic) engine=Memory;
+insert into table test select * from numbers(5);
+alter table test modify column d Dynamic(max_types=1);
+select d.UInt64 from test settings allow_experimental_analyzer=1;
+select d.UInt64 from test settings allow_experimental_analyzer=1;
```

Comment:
```suggestion
select d.String from test settings allow_experimental_analyzer=1;
select d.UInt64 from test settings allow_experimental_analyzer=0;
select d.String from test settings allow_experimental_analyzer=0;
```

First of all I don't see why the second, same read is necessary. On the other hand, we can extend the tests to cover a bit more of the code very easily. Therefore in my opinion these changes make sense. Please correct me if I am wrong.

Reply:
The second actually should have `allow_experimental_analyzer=0` to test the same querry with and without analyzer. It was a typo